### PR TITLE
[Metricbeat] Add Overview dashboard to Tomcat module

### PR DESCRIPTION
**Pull request by sayden**
Overview dashboard for Tomcat metricbeat module

~https://user-images.githubusercontent.com/4249331/66661664-ab29fe80-ec47-11e9-852d-2d380b29ac20.png~

~https://user-images.githubusercontent.com/4249331/66813821-c5a8f400-ef35-11e9-8381-8cf316f9279d.png~

~!https://user-images.githubusercontent.com/4249331/67019835-8e853f00-f0fd-11e9-9efd-ece11a96fa9c.png~

image

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -413,7 +413,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add cloud.account.id and cloud.account.name into events from aws module. {issue}13551[13551] {pull}13558[13558]
 - Add `metrics_path` as known hint for autodiscovery {pull}13996[13996]
 - Leverage KUBECONFIG when creating k8s client. {pull}13916[13916]
-
+- Add Tomcat overview dashboard {pull}14026[14026]
```
Comment (jsoriano) [inline CHANGELOG.next.asciidoc]: Nit. Keep the empty line below.
```suggestion
- Add Tomcat overview dashboard {pull}14026[14026]

```

**Comment by sayden** [comment]
jenkins, test this

File: x-pack/metricbeat/module/tomcat/_meta/kibana/7/dashboard/Metricbeat-Tomcat-overview.json [hunk 2]
```diff
@@ -0,0 +1,1018 @@
...
+            "gridData": {
+              "h": 9,
+              "i": "6",
+              "w": 23,
+              "x": 25,
+              "y": 0
+            },
+            "panelIndex": "6",
+            "panelRefName": "panel_4",
+            "title": "Derivative of total requests",
```
Comment (jsoriano) [inline x-pack/metricbeat/module/tomcat/_meta/kibana/7/dashboard/Metricbeat-Tomcat-overview.json]: No need to mention in the visible titles that this is a derivative.

**Comment by andresrc** [comment]
is there anything blocking this?

ping @sayden @jsoriano

**Comment by andresrc** [comment]
Part of #13860

**Comment by jsoriano** [comment]
I wouldn't say it is blocking but there is an unaddressed comment, I think we should title the visualizations with the value they are representing, not with the calculation made. e.g. something like `Total requests` or `Requests over time`, or `Requests per second`, instead of `Derivative of total requests`.

**Comment by sayden** [comment]
Ok, title have been changed.

**Comment by jsoriano** [comment]
> Ok, titles have been changed.

Thanks! Could you regenerate the screenshot?

**Comment by sayden** [comment]
Screenshot updated

**Pull request merged**